packages/react-core/src/next/components/Modal/ModalBoxBody.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/ModalBox/modal-box';
+
+export interface ModalBoxBodyProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the modal box body. */
+  children?: React.ReactNode;
+  /** Additional classes added to the modal box body. */
+  className?: string;
+  /** Accessible label applied to the modal box body. This should be used to communicate
+   * important information about the modal box body div element if needed, such as that it
+   * is scrollable.
+   */
+  'aria-label'?: string;
+  /** Accessible role applied to the modal box body. This will default to "region" if the
+   * bodyAriaLabel property is passed in. Set to a more appropriate role as applicable
+   * based on the modal content and context.
+   */
+  'aria-role'?: string;
+  /** Id of the modal box body. This should mathc hte modal box header descriptorId????? */
+  id?: string;
```

Comment:
I feel like we could remove this prop, as ideally the Modal's `aria-describedby` should point to the ModalBoxDescription or a similar description element (like in the Wizard example, we're pointing to the `pf-v5-c-wizard__description` element)

If the ModalBoxBody contains a lot of content, then the aria-description would end up being pretty verbose.